tests/test_services/test_morpho_img.py

```diff
@@ -7,7 +7,6 @@
 from unittest.mock import patch
 from api.services.morpho_img import generate_morphology_image
 from tests.fixtures.utils import load_content
-from tests.fixtures.nexus import morphology_content_url, access_token
```

Comment:
@bilalesi I think this removal breaks the tests

Reply:
fixed